sbc_switzerland/models/correspondence.py

```diff
@@ -130,7 +130,7 @@ def process_letter(self):
                 if intro_letter in letter.communication_type_ids and not \
                         letter.sponsorship_id.send_introduction_letter:
                     continue
-                if super(Correspondence, letter).process_letter():
+                if super().process_letter():
```

Comment:
This should be reverted, because the super was not called on the whole recordset.